src/markdown-pages/add-ons/local-path-provisioner.md

```diff
@@ -0,0 +1,21 @@
+---
+path: "/docs/add-ons/local-path-provisioner"
+date: "2022-06-08"
+linktitle: "Local Path Provisioner Add-On"
+weight: 46
+title: "Local Path Provisioner Add-On"
+addOn: "localPathProvisioner"
+---
+
+[Local Path Provisioner](https://github.com/rancher/local-path-provisioner/) is a minimal PVC provisioner that exposes node-local storage.
+
+Local Path Provisioner uses the `/opt/local-path-provisioner` directory for storage on all nodes.
+This directory should have enough space to hold a complete copy of every PersistentVolumeClaim that will be on a given node.
```

Comment:
This begs the question, how much is enough ? is it possible if there is something we can recommend or can we point to Ranchers' doc?

Reply:
How can we help the user determine how much is enough? We should think about what our typical vendor may need or help guide them on how to figure it out. I would suspect that backups highly impact this (but do not know for sure - snaps that I am used to usually load a delta to the side so for example if it is 50GB and you snap, then it is 50GB + any additional writes to the new snapshot)

Reply:
@laverya as trivial as it sounds, whatever you are saying above has to go to docs i feel!

Reply:
updated